infrastructure/terraform/components/dl/module_ttl_poll_lambda.tf

```
@@ -0,0 +1,76 @@
+module "ttl_poll_lambda" {
+  source = "https://github.com/NHSDigital/nhs-notify-shared-modules/releases/download/v2.0.20/terraform-lambda.zip"
```

Again, this isn't the latest version:
```suggestion
  source = "https://github.com/NHSDigital/nhs-notify-shared-modules/releases/download/v2.0.24/terraform-lambda.zip"
```